### PR TITLE
added: grid component

### PR DESCRIPTION
**Pull request by sharmayatendra**
# Grids

## Open for review

image

### CODE REVIEW

**Review by vivek-guda — COMMENTED** [review]
Looks good, Nice work!

File: components/Grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
+:root {
+    --grid-item-bg-color: rgb(212, 243, 212);
+}
+.grid-two-items {
+    border: 1px solid var(--primary-color);
+    display: grid;
+    gap: 10px;
+    grid-template-columns: repeat(2, 1fr);
```
Comment (HrishikeshMahalle) [inline components/Grid/grid.css:8]: Try to use rem in Gaps, helps in responsiveness

File: components/Grid/grid.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
+:root {
+    --grid-item-bg-color: rgb(212, 243, 212);
+}
```
Comment (HrishikeshMahalle) [inline components/Grid/grid.css:3]: Hex Values are Better, but no worries

**Review by HrishikeshMahalle — APPROVED** [review]
Not much big changes to look after. Else looks Fine. Good Job!

**Review by RaghavendraaReddy — APPROVED** [review]

File: components/Grid/grid.css (hunk 1 above)
Comment (sharmayatendra) [reply components/Grid/grid.css:8]: As told in the lecture we can use the "px" unit for values up to 10px.

File: components/Grid/grid.css (hunk 2 above)
Comment (sharmayatendra) [reply components/Grid/grid.css:3]: I used RGB values everywhere. Thanks for the suggestions though.

**Pull request merged**